backend/cmd/zeevision/main.go

```diff
@@ -83,16 +88,16 @@ func main() {
 		err = kafkaConsumer.ConsumeTopic(0, topic)
 		if err != nil {
 			// TODO: error handling
-			panic(err)
+			zap.L().Panic("Failed consuming", zap.String("topic", topic), zap.Error(err))
 		}
 	}
 
 	server, err := endpoint.NewFromEnv(fetcher)
 	if err != nil {
-		log.Fatal(err)
+		zap.L().Fatal("Failed creating a new endpoint from environment variables")
 	}
 
 	if err := server.Run(); err != nil {
-		log.Fatal(err)
+		zap.L().Fatal("Errors happens when trying to run server")
```

Comment:
You can rephrase this to just `Server error`, since it will be printed for all errors during operation. Also include the causing error in the log.